docs/pages/reference/api/getting-started.mdx

```diff
@@ -69,31 +75,43 @@ func main() {
 	ctx := context.Background()
 
 	clt, err := client.New(ctx, client.Config{
+		Addrs: []string{
```

Comment:
`Addrs` aren't necessary when you use `client.LoadProfile`. The proxy address used to login via `tsh`, in this case `tele.example.com:443`, will be used to retrieve the auth address. No harm in listing them if you think it serves a purpose, just wanted to point it out.

Reply:
I just tested with and without the `Addrs` array. All other things, including `-proxy` flag being left the same:

With it:
```
go run main.go

2021/05/24 20:08:41 Example success!
2021/05/24 20:08:41 Example server response: {publictest.go-teleport.com 6.1.5 Kubernetes:true App:true DB:true  publictest.go-teleport.com:443 {}  %!s(int32=0)}
2021/05/24 20:08:41 Server version: 6.1.5
```

Without it:
```
go run main.go
2021/05/24 20:10:41 failed to create client: all auth methods failed
	context deadline exceeded
exit status 1
```

Reply:
The Teleport server itself generates the following message:

```
ERRO [PROXY:SER] "proxy2021/05/24 20:08:41 http: TLS handshake error from 3.140.251.104:43494: acme can't get a cert for domain teleport.cluster.local, add it to the proxy_service.public_addr, or use one of the domains: publictest.go-teleport.com\n" utils/cli.go:304
```

Reply:
Hmm that should work, maybe the proxy address you used to log in is different from the ones you listed in `Addrs`.

If you have time, could you make a bug ticket? Include your steps for logging in and the contents of `~/.tsh/current-profile` and `~/.tsh/<current-profile>.yaml`.